.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Mocha Tests",
+            "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
+            "args": [
+                "--compilers", "js:babel-core/register",
+                "--require", "babel-polyfill",
+                "${workspaceRoot}/test/**/*.spec.js",
+                "--debug"
+            ]
+        }
+    ]
+}
```

Comment:
Are these .vscode files required for anyone to build/test the project? We need to make sure people are able to do all those things without vscode. I'm OK with keeping these config files if they're purely optional helpers if the builder happens to use vscode (which I do).